### PR TITLE
Initial commit for the temperature

### PR DESCRIPTION
**Pull request by dinadeljanin**
Barebones component for the temperature. 

Some things to note:
- Positioning will be applied once it's inside a diagnostic component, but for right now,  it sits at the top left by default.
- Font and container size does adjust for different mobile sizes.
- Left a placeholder for the temperature.
- Working on translations as well.

### CODE REVIEW

File: client/src/components/Temperature.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      <Card>
+        <Card.Body>
+            <Card.Title id="Temperature-temp">
+              98.8F
+            </Card.Title>
+            <Card.Subtitle id="Temperature-avgTempText">
+              Average Temp
+            </Card.Subtitle>
+        </Card.Body>
+      </Card>
```
Comment (SomeMoosery) [inline client/src/components/Temperature.js]: Unless I'm missing something, do we necessarily need a `Card` here? I like the fact that the temperature and text "Average Temp" are tightly coupled like this, but I'm wondering if there's a way to get rid of the border? I feel that in the actual UI, that could look a little out of place.

File: client/src/css/NavBar.css [hunk 2]
```diff
@@ -83,7 +83,6 @@
 @media only screen and (max-width: 414px) {
   .nav {
     height: 12vh;
-    max-width: 414px;
```
Comment (SomeMoosery) [inline client/src/css/NavBar.css]: Any reason you removed this in the NavBar?

File: client/src/components/Temperature.js (hunk 1 above)
Comment (dinadeljanin) [reply client/src/components/Temperature.js]: I can do that-- I wanted to use as much react-bootstrap as i could before i made it a div, but i could switch it over to a div. 

I did notice the rounded border for the card. I wonder-- and this is going much further into the design-- perhaps even a shadow could be added to the card border itself to give it some depth.

File: client/src/css/NavBar.css (hunk 2 above)
Comment (dinadeljanin) [reply client/src/css/NavBar.css]: I remember an issue with the navbar extending far past the viewport width, and that's why that was there. I forgot to remove it in the earlier commit.

File: client/src/components/Temperature.js (hunk 1 above)
Comment (SomeMoosery) [reply client/src/components/Temperature.js]: I'll defer that question to @BrianHHough as he's been on top of the design aspect of this, but yeah I'd say either removing the border entirely or adding a showing would be the two routes here 👍

File: client/src/css/NavBar.css (hunk 2 above)
Comment (SomeMoosery) [reply client/src/css/NavBar.css]: Makes sense, Feel free to resolve this when you push your next commit removing it!

File: client/src/components/Temperature.js (hunk 1 above)
Comment (dinadeljanin) [reply client/src/components/Temperature.js]: @SomeMoosery Hey I removed the border and border radius!

Comment (SomeMoosery) [reply client/src/components/Temperature.js]: Great!

**Pull request merged**